package.json

```diff
@@ -73,6 +73,7 @@
 		"test:e2e:mainnet:ws": "./scripts/test-runner.sh mainnet ws",
 		"test:e2e:sepolia:http": "./scripts/test-runner.sh sepolia http",
 		"test:e2e:sepolia:ws": "./scripts/test-runner.sh sepolia ws",
+		"test:manual":"node ./packages/web3/test/stress/manual/nodejs_test/connection.js",
```

Comment:
Plz add one script command pattern for all stress tests ( coordinate with oleksii for it ) aligned with https://github.com/web3/web3.js/pull/6595/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R87